app/assets/javascripts/application.js.erb

```diff
@@ -176,8 +177,9 @@ function display_node_rows( d ) {
 function display_enhanced_job_data( d ) {
 
     if (is_defined(d.error)) {
-        return      '<div><tr><td>'+ d.error +'</td></tr></div>';
+        return      '<div><tr><td><div class="alert alert-warning">No job details because job has already left the queue. </div></td></tr></div>';
```

Comment:
Oh I see. So this is happening because we are rescuing from StandardError instead of `PBS::UnkjobidError` because we are using `pbs` directly.

So the solution is perhaps to do this:

```
rescue PBS::UnkjobidError
  { name: pbsid, error: "No job details because job has already left the queue." , status: "C" }
rescue => e
  { name: pbsid, error: "No job details available." }
end
```

Of course this edge case would be covered when we switch to ood_job, or addressed differently, so we could delay this change till then.